src/main/java/io/cdap/e2e/pages/locators/CdfNameSpaceAdminLocators.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.locators;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * Represents CdfNameSpaceAdminLocators
+ */
+public class CdfNameSpaceAdminLocators {
+
+  @FindBy(how = How.XPATH, using = "//div[contains(text(),'Namespace Admin')]")
+  public static WebElement namespaceAdminMenu;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='navbar-hamburger-icon']")
+  public static WebElement hamburgerMenu;
+
+  public static WebElement nameSpaceModules(String module, String nameSpace) {
+    String path = "//*[@href=\"/cdap/ns/" + nameSpace + "/details/" + module + "\"]";
+    return SeleniumDriver.getDriver().findElement(By.xpath(path));
+  }
+
+  public static WebElement createProfile(String nameSpace) {
+    String path = "//*[@href=\"/cdap/ns/" + nameSpace + "/profiles/create\"]";
+    return SeleniumDriver.getDriver().findElement(By.xpath(path));
+  }
+
+
+  public static WebElement switchNameSpace(String nameSpace) {
+    String path = "//*[@href=\"/cdap/ns/'" + nameSpace + "'\"]";
+    return SeleniumDriver.getDriver().findElement(By.xpath(path));
+  }
+
+  @FindBy(how = How.XPATH, using = "//*[@class='namespace-dropdown']")
+  public static WebElement namespaceDropdown;
+
+  @FindBy(how = How.XPATH, using = "//div[contains(text(),'Add Namespace')]")
+  public static WebElement addNamespace;
+
+  @FindBy(how = How.XPATH, using = "//div[@data-testid='create-namespace-name']")
+  public static WebElement namespaceName;
+
+  @FindBy(how = How.XPATH, using = "//input[@placeholder='Namespace description' and @type='text']")
+  public static WebElement namespaceDescription;
+
+  @FindBy(how = How.XPATH, using = "//button[@data-cy='wizard-next-btn']")
+  public static WebElement clickNext;
+
+  @FindBy(how = How.XPATH, using = "//button[@data-cy='wizard-finish-btn']")
+  public static WebElement clickFinish;
```

Comment:
Same applies here.

Reply:
Done , changed it to clickOnFinishButton